test/17.js

```diff
@@ -0,0 +1,18 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/17').solution;
+// solution = require('./yourSolution').solution;
+
+describe('squareMatrix', () => {
+  it('should return true', () => {
```

Comment:
Please use better messaging. `should be true when input is 3x3`, `should be false when input is 2x3`, `should be true when input is 2x2`